state-sync/storage-service/server/src/lib.rs

```diff
@@ -20,7 +26,10 @@ mod tests;
 
 // TODO(joshlind): make these configurable.
 /// Storage server constants.
+pub const MAX_EPOCH_CHUNK_SIZE: u64 = 1000;
 pub const MAX_TRANSACTION_CHUNK_SIZE: u64 = 1000;
+pub const MAX_TRANSACTION_OUTPUT_CHUNK_SIZE: u64 = 1000;
+pub const MAX_ACCOUNT_STATES_CHUNK_SIZE: u64 = 1000;
```

Comment:
random tangent: I wonder how we should handle a case where a chunk of accounts/txns/txn outputs exceeds 10 MiB (DiemNet max message size)? This was something we punted on when implementing Mempool and StateSync v1, but we probably need a better strategy going forward.

Today we effectively just assume that all possible chunks are <= 10 MiB. If that's not the case, our current approach would cause the Data Fetcher to stall if it hits a chunk that can't be transported over DiemNet. I guess we could either just choose conservative max chunk sizes so it's not a problem, or come up with something more sophisticated.

we could do some work to expose that network error case, then a Data Fetcher could retry that chunk in smaller chunks if it sees that error?

Reply:
Yeah, this is a good question! My currently feelings are that we'll start by choosing conservative values for these chunks (so we don't hit the limit for now). Later on, we can think a bit more broadly about this, e.g.,  either expose the error (like you say) or try to wrap this problem up in a bigger effort that makes applications have a more wholistic view of the network in general. But, let's punt on it for now and once we have an MVP, we can look at it. FYI: I suspect there's gonna be other issues in this area going forward and many of them we'll probably only be able to identify once we have an end-to-end MVP.